src/OrbitGl/CMakeLists.txt

```diff
@@ -60,15 +60,14 @@ target_sources(
          ScopeTree.h
          ShortenStringWithEllipsis.h
          StatusListener.h
-         TextBox.h
+        ../ClientData/include/ClientData/TextBox.h
```

Comment:
Shouldn't you include the ClientData project instead?

Reply:
right - I was looking for this one and missed somehow.